### PR TITLE
Feature/cannot import into last test run

### PR DESCRIPTION
**Pull request by VladislavKostyukevich**
# PR Details

Fix Issue related to API User permissions that blocks API Import with addToLastTestRun option

## Related Issue

#29 

## How Has This Been Tested

1. Import file via API with addToLastTestRun option

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[0904/154646.082:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
```
Comment (knysh) [inline debug.log]: is it really necessary in repository?

File: e2e/pages/import.po/index.ts [hunk 2]
```diff
@@ -70,17 +82,28 @@ export class Import extends BasePage {
...
-      return oldLatestTestRunId !== actualLatestTestRunId && isFinished;
-    }, browser.allScriptsTimeout).catch(() => false);
+      try {
+        const actualLatestTestRundate = await this.getLatestImportedTestRunDate();
+        const isFinished = await this.isLatestImportFinished();
+        return lastTestRunDate < actualLatestTestRundate && isFinished;
+      } catch (error) {
+        return false;
+      }
+    }, 5, 3000);
```
Comment (knysh) [inline e2e/pages/import.po/index.ts]: is it possible to move timeouts to config file?

File: debug.log (hunk 1 above)
Comment (VladislavKostyukevich) [reply debug.log]: removed in next pr

File: e2e/pages/import.po/index.ts (hunk 2 above)
Comment (VladislavKostyukevich) [reply e2e/pages/import.po/index.ts]: this one is ralated to only import time, not sure if I want to store in separately from import page object

**Pull request merged**